README.md

```diff
@@ -36,12 +36,13 @@ Places to ask questions and find other community resources.
 
 ### Data
 
-- [GTFS Data Exchange](http://www.gtfs-data-exchange.com/agencies) - Links to many transit agency GTFS datasets.
+- [Transitland](https://transit.land/) - A community-edited data service aggregating transit networks across metropolitan and rural areas around the world. Currently under development.
+
```

Comment:
I'd remove the "currently under development".  It's now live and in production.  I'd also add the sentence  about "links to many transit agency GTFS datasets" here as well as well.  IMHO Transitland is effectively replacing GTFS Data Exchange.